ui/src/app/pages/data/data-set/genomic-extraction-modal.tsx

```diff
@@ -25,7 +39,31 @@ interface Props {
 export const GenomicExtractionModal = ({
     dataSet, workspaceNamespace, workspaceFirecloudName, closeFunction, title, cancelText, confirmText}: Props) => {
   const [error, setError] = useState(false);
-  const [loading, setLoading] = useState(false);
+  const [loading, setLoading] = useState(true);
+
+  const genomicExtractions = useStore(genomicExtractionStore);
+  const extractsForWorkspace = genomicExtractions && genomicExtractions[workspaceNamespace] || [];
+  const extractsForDataset = fp.filter((extract: GenomicExtractionJob) => extract.datasetName === dataSet.name, extractsForWorkspace);
+
+  useEffect(() => {
+    if (!(workspaceNamespace in genomicExtractions)) {
```

Comment:
I think it's fine to just call this API endpoint on load. On the off chance that the store is a bit out of date or another user in the workspace launched a job.

It'll add some load time to the modal but I think that's OK given this workflow won't often be used and they'll be waiting for hours afterwards anyways.

Reply:
rats, i once again did not think about the other-user case. thank you!

Reply:
did u mean to remove this conditional?